core/src/core-plugin-definitions.ts

```diff
@@ -408,6 +408,10 @@ export interface DevicePlugin extends Plugin {
 export type OperatingSystem = 'ios' | 'android' | 'windows' | 'mac' | 'unknown';
 
 export interface DeviceInfo {
+  /**
+   * The name of the device. For example, "John's iPhone"
+   */
+  name: string;
```

Comment:
Since name is iOS only, it should be optional (add the ? at the end), that means it might or might not be returned.
```suggestion
  name?: string;
```